explainer.md

```diff
@@ -622,19 +614,24 @@ partial interface Navigator {
   readonly attribute VR vr;
 };
 
-[SecureContext, Exposed=Window] interface VR : EventTarget {
-  attribute EventHandler ondeviceconnect;
-  attribute EventHandler ondevicedisconnect;
+dictionary VRDeviceFilter {
+  boolean exclusive = false;
+};
+
+dictionary VRDeviceRequestOptions {
+  required sequence<VRDeviceFilter> filters;
+};
 
-  Promise<sequence<VRDevice>> getDevices();
+[SecureContext, Exposed=Window] interface VR : EventTarget {
+  attribute EventHandler ondeviceschanged;
+  Promise<VRDevice> requestDevice(optional VRDeviceRequestOptions options);
```

Comment:
to avoid adding the new dictionary interface, `VRDeviceRequestOptions`, can this instead be a plain `object`? (inspiration: [`navigator.permissions.query`](https://w3c.github.io/permissions/#permissions-interface)) I seem to recall @annevk's making a similar suggestion in prior discussions.

Reply:
I'd try to avoid `object` as it means you have to write your own binding logic in prose.

Reply:
gotcha, makes sense. I assume that was the rationale behind this issue you filed: https://github.com/w3c/permissions/issues/158

Reply:
No, that had a different reason (and doesn't seem to be going anywhere).